docker/package/model.py

```diff
@@ -232,7 +232,8 @@ def fetch_sources(self, out_dir, binaries_dir=None):
 
```

Comment:
Pleas also reset the `letter_version` for the baking package here in a separate commit (it doesn't need to be `"a"` for a new release)